OpenTabletDriver.Web/Views/Wiki/Development/Configurations.cshtml

```diff
@@ -0,0 +1,324 @@
+            <td>Product ID</td>
+            <td>
+                <code>ushort</code>
+            </td>
+            <td class="text-start">
+                A vendor-defined identifier for the device. This can be used to identify a device,
+                however this can be unreliable as it depends on the vendor defining it different
+                for all of their devices.
+            </td>
+        </tr>
+        <tr>
+            <td>Input Report Length</td>
+            <td>
+                <code>ushort</code>
+            </td>
+            <td class="text-start">
+                The length of input reports from the device. This is always required as report parsers
+                expect a specific report length.
+            </td>
+        </tr>
+        <tr>
+            <td>Output Report Length</td>
+            <td>
+                <code>ushort</code>
+            </td>
+            <td class="text-start">
+                The length of output reports to the device endpoint. This is not always required,
+                however it does help make the detection more precise.
+            </td>
+        </tr>
+        <tr>
+            <td>Report Parser</td>
+            <td>
+                <code>string</code>
+            </td>
+            <td class="text-start">
+                The parser that will read and convert all tablet report data into a format that
+                OpenTabletDriver understands. This is the full namespace and the class name.
+                <br/>
+                <small class="text-muted ms-3">
+                    Note: All supported vendor-specific report parsers can be found
+                    <a class="link-light" href="@repoUrl/tree/HEAD/OpenTabletDriver.Configurations/Parsers">here</a>
+                </small>
+            </td>
+        </tr>
+        <tr>
+            <td>Feature Initialization Report</td>
+            <td>
+                <code>List&lt;byte[]&gt;</code>
+            </td>
+            <td class="text-start">
+                A list of feature reports to be sent to the device to perform the device's initialization sequence.
+            </td>
+        </tr>
+        <tr>
+            <td>Output Initialization Report</td>
+            <td>
+                <code>List&lt;byte[]&gt;</code>
+            </td>
+            <td class="text-start">
+                A list of output reports to be sent to the device to perform the device's initialization sequence.
+            </td>
+        </tr>
+        <tr>
+            <td>Device Strings</td>
+            <td>
+                <code>Dictionary&lt;byte, string&gt;</code>
+            </td>
+            <td class="text-start">
+                A list of regular expressions for specific indexes of device strings contained within the device's
+                firmware. This is optional however is used commonly to improve detection precision. Device strings
+                can be retreived using the device string reader.
```

Comment:
This is pretty messy. Not sure how to reword it better.

Reply:
I would just add a short example of what device strings look like, (eg an output of a dump done) so people can see how the regexes apply.

Reply:
> I would just add a short example of what device strings look like, (eg an output of a dump done) so people can see how the regexes apply.

I feel that this should be done in its own section rather than in this overview.

Reply:
Oh yeah even better if you plan to have more sections delving deeper, lg2m